vuepress/docs/next/docs/curate/bundle-comparison.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Bundle Transition to 7.1 
+
+Entando 7.1 implements a more modular approach to building apps with the **ent bundle** CLI orchestrating the management of project bundles. The Entando bundle was restructured, its processes decoupled by component type, and the publishing transport method was streamlined. With these changes, Entando Bundles went from git-based to Docker-based OCI bundles, and this page clarifies the differences between the two, along with the improvements that were made with the new release.
+
+Some of the major improvements with the docker-based bundles are:
+* Simplified bundle directory structure
+* Just one source git repository required with Docker image repositories created automatically as needed
+* Optimized file storage and transfer using a layered file-system, beneficial for large projects
+* Standardized bundle address format
+* A decoupled build and deployment process which is easier to understand, modify, and troubleshoot 
+
+For specifications on docker-based bundles, see the [Bundle Details](ecr-bundle-details.md) page. 
+### Bundle Comparison Table
+|Attribute | [git-based bundles](../../../v7.0/docs/curate/ecr-bundle-details.md)| [docker-based bundles](ecr-bundle-details.md)|
+| :- | :-------------------- | :---------------------  |
+|`descriptorVersion`| v1| v5
+| Bundle specifications |Defined with descriptor files | Built with [ent bundle CLI](../getting-started/ent-bundle.md) tool
+| Versioning |Bundle version set by git tags |Bundle version set by Docker Tags. For MFEs, the version defaults to 1.0.0 unless set in the [entando.json](ecr-bundle-details.md). 
```

Comment:
"Git tags" and "Docker tags" - uppercase for feature/platform & lowercase "tag"

Reply:
will get both in next round